timeless/reservations/views.py

```diff
@@ -18,10 +18,6 @@
 class SettingsList(views.ListView):
     """
     List view set for Reservation Settings
-    @todo #186:30min For SettingsListView, SettingsCreateView,
-     SettingsDetailView, SettingsDeleteView create correct templates
```

Comment:
@paulodamaso what about the detail view and the delete view? Shouldn't you leave a puzzle for them?